app/build.gradle.kts

```diff
@@ -111,19 +111,21 @@ dependencies {
 
 android {
     namespace = "app.revanced.manager"
-    compileSdk = 35
-    buildToolsVersion = "35.0.1"
+    compileSdk = 36
+    buildToolsVersion = "36.0.0"
```

Comment:
Is this even necessary now?

Reply:
we're bumping the target and compile sdk to 36, so it would also be appropriate to update the build tools to 36 also. (even though using 35.0.0 or 36.1.0 or 34.0.0 is fine really but it doesn't make sense.)